### DISCUSSION TITLE
Stratocumulus bias in CESM3

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#527] [Discussion about coupled runs]
In preparation of the CAM7 meeting, here are the current runs (if you want to run your scripts):
```
L58: /glade/derecho/scratch/hannay/archive/b.e23_alpha16g.BLT1850.ne30_t232.082b 
L32 :/glade/derecho/scratch/hannay/archive/b.e23_alpha16g.BLT1850.ne30_t232_L32.082b
L42: /glade/derecho/scratch/hannay/archive/b.e23_alpha16g.BLT1850.ne30_t232_L42.082b
```
Note the L42 is a grid that Julio created yesterday with the L58-level in the PBL and L32-level above PBL. This run is not completed yet.

I also have a 2-year high frequency version of 82b to look at the daily cycle. 
```
/glade/derecho/scratch/hannay/archive/b.e23_alpha16g.BLT1850.ne30_t232_daily.082b
```

**Comment by swrneale** [comment]
@cecilehannay It occurs to me that another sensitivity we might want to test is the vertical length scale of the parcel mixing in the ZM parcel mods. It's currently 0.5*PBL-height (I think), but we could try 0.25*PBL height, which will increase sensitivity to near-surface resolution a bit. We should test this anyway for it's impact on other tropical precip. features. It is not a namelist parameter, it is just a fixed parameter in zm_conv.F90 [parcel_hscale  = 0.5_r8] just change it to 0.25_r8.   I have evidence that the 0.5 value may be overdoing the parcel dilution.
